lightning/src/ln/channelmanager.rs

```diff
@@ -5276,6 +5276,16 @@ impl<'a, Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref>
 				None => continue,
 			}
 		}
+		if forward_htlcs_count > 0 {
+			// If we have pending HTLCs to forward, assume we either dropped a
+			// `PendingHTLCsForwardable` or the user received it but never processed it as they
+			// shut down before the timer hit. Either way, set the time_forwardable to a small
+			// constant as enough time has likely passed that we should simply handle the forwards
+			// now, or at least after the user gets a chance to reconnect to our peers.
+			pending_events_read.push(events::Event::PendingHTLCsForwardable {
+				time_forwardable: Duration::from_secs(5),
```

Comment:
5 seconds seems like a lot of payment latency (imagining standing at a cashier and waiting for the payment to complete..)

Would, say, 200 milliseconds provide a decent amount of privacy instead? :thinking:

Reply:
On reload its less about privacy, during runtime its handled by `MIN_HTLC_RELAY_HOLDING_CELL_MILLIS` in channelmanager which is currently 100ms. On reload we're more concerned about how long it takes the application to reconnect to all our peers and make sure the channels are up when we forward/claim any pending HTLCs. I'm not sure how much we care about latency - we just reloaded, presumably we've been offline for a while (I guess its possible something went wrong and someone just restarted the application? dunno), vs reliability here.

Reply:
Alright, just noticed that we also encourage users to wait for a random interval in `[time_forwardable, time_forwadable * 5]` so as long as we're fine with up to 25 seconds I'm good with this as a first iteration

Reply:
Hmm, good point, 25 seconds is probably too much, maybe we should set it to 1 or 2 seconds and just hope no one is using tor and gets unlucky? I guess 2 seconds is probably fine even for that, but 1 is pretty quick.

Reply:
2 seconds SGTM